editor/components/block-list/layout.js

```diff
@@ -215,6 +216,7 @@ class BlockListLayout extends Component {
 
 		return (
 			<BlockSelectionClearer className={ classes }>
+				{ !! blockUIDs.length && <BlockInsertionPoint /> }
```

Comment:
This needed to be passed `rootUID`. When trying to insert a block before the first block in a Columns block, it appears in the top-level instead.